extensions/docker/package.json

```diff
@@ -25,5 +25,27 @@
     "mkdirp": "^3.0.1",
     "vite": "^6.0.7",
     "vitest": "^2.1.6"
+  },
+  "contributes": {
+    "commands": [
+      {
+        "command": "docker.cli.context.onChange",
+        "title": "Callback for Docker CLI context change",
+        "category": "Docker",
+        "enablement": "false"
+      }
+    ],
+    "configuration": {
+      "title": "Docker",
+      "properties": {
+        "docker.cli.context": {
+          "type": "string",
+          "enum": [""],
```

Comment:
Are we forced to defined an empty string here ? Since this is set through the context

Reply:
it's to tell that the property is an enum

Reply:
could be an empty array but I think I had error reported but could be a false positive error feedback with empty